bin/http-server

```diff
@@ -80,6 +82,54 @@ else if (colors) {
     request: function () {}
   };
 }
+if (superverbose) {
+  logger = {
+    info: console.log,
+    request: function (req, res, error) {
+      var date = utc ? new Date().toUTCString() : new Date();
+      if (error) {
```

Comment:
Should we be showing all this info if there was an error, or if there was no error? Was the logic accidentally flipped here?